### PR TITLE
new package: libxdg-basedir 1.2.0

### PR DESCRIPTION
**Pull request by a-schaefers**
## Checklist

- [x] Latest upstream version.
- [x] I agree to maintain this package (**required**).
- [x] I agree to notify KISS if I can no longer maintain this package (**required**).

### CODE REVIEW

File: community/libxdg-basedir/build [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+./configure --prefix=/usr \
+            --enable-unicode-properties \
+            --enable-pcre16 \
+            --enable-pcre32 \
+            --enable-jit \
+            --enable-pcregrep-libz \
+            --enable-pcregrep-libbz2 \
+            --enable-pcretest-libreadline
+
+make DESTDIR="$1" install
```
Comment (dylanaraps) [inline community/libxdg-basedir/build]: Missing standalone `make`?

File: community/libxdg-basedir/build [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#!/bin/sh -e
+
+./configure --prefix=/usr \
+            --enable-unicode-properties \
+            --enable-pcre16 \
+            --enable-pcre32 \
+            --enable-jit \
+            --enable-pcregrep-libz \
+            --enable-pcregrep-libbz2 \
+            --enable-pcretest-libreadline
```
Comment (dylanaraps) [inline community/libxdg-basedir/build]: Fix indentation and move `--prefix` onto a new line.

**Comment by dylanaraps** [comment]
Is this missing a dependency on `pcre`?

File: community/libxdg-basedir/build (hunk 1 above)
Comment (a-schaefers) [reply community/libxdg-basedir/build]: no

**Comment by a-schaefers** [comment]
> Is this missing a dependency on `pcre`?

no

**Pull request merged**